src/Nethermind/Nethermind.Core/BlockHeader.cs

```diff
@@ -47,6 +49,20 @@ public class BlockHeader
             GasLimit = gasLimit;
             Timestamp = timestamp;
             ExtraData = extraData;
+            ExcessDataGas = excessDataGas;
+            ParentExcessDataGas = parentExcessDataGas;
```

Comment:
do we need ParentExcessDataGas here? Is ParentHash not enough? It sounds to be the similar situation to EIP1559. Please, check BaseFeeCalculator.cs class

Reply:
I'll remove it now and return in the gas calculations request, if it will be still needed. It's hard to show for now why it's useful